src/python/grpcio/grpc/_server.py

```diff
@@ -945,49 +1069,60 @@ def _validate_generic_rpc_handlers(generic_rpc_handlers):
                 'not have "service" method!'.format(generic_rpc_handler))
 
 
-def _augment_options(base_options, compression):
+def _augment_options(
+    base_options: Optional[Sequence[ChannelArgumentType]],
```

Comment:
`base_options` cannot be `None` [at this layer](https://github.com/grpc/grpc/blob/c20455e8363836aae4750d4325704496a5a3c278/src/python/grpcio/grpc/__init__.py#L2070).